.github/workflows/benchmark-flow.yml

```diff
@@ -37,7 +37,7 @@ env:
 
 jobs:
   benchmark-steps:
-    runs-on: ubuntu-latest
+    runs-on: ${{ vars.RUNS_ON_UBUNTU_LATEST }}
```

Comment:
This sounds like a boolean variable, yet I assume this variable will get the Ubuntu version we will run on, correct?
If so, rename to something more explicit like `UBUNTU_VERSION`